### DISCUSSION TITLE
Angle of Arrival Bluetooth Conectionless Example Channels

### DISCUSSION THREAD

**Discussion by vChavezB** [zephyrproject-rtos/zephyr#57663] [General]
I am testing the connectionless rx example from here

https://github.com/zephyrproject-rtos/zephyr/tree/main/samples/bluetooth/direction_finding_connectionless_rx

And when I am getting the I/Q reports I get sometimes in the report struct `bt_df_per_adv_sync_iq_samples_report ` that the channel idx are outside the adv. channels (37,38,39). E.g., 20,30,11,14

I was just wondering if this is normal and if the CTE during periodic advertising is transmitted over channels that are not the adv. channels?

Edit: Based on docs from silabs the periodic adv. uses data channels. 

https://docs.silabs.com/bluetooth/latest/general/adv-and-scanning/periodic-adv-bt5

So I might rephrase this, is it possible to select which channels I want the periodic adv. to be sent ?

**Comment by hermabe** [comment]
You could use the channel map to inform the controller about used and unused channels, but the controller is free to change the channel map itself. I do not know the implementation details here.

**Reply by vChavezB** [reply]
oh ok I see, I was asking since I have seen in papers that use BLE AoA they state that use a specific adv. channel for their experiments. 

From what I see in the Bluetooth spec. the channels are selected according to algorithm no. 1 and no. 2 , so perhaps the authors of the different papers just filter the I/Q reports according to the channel they want to work (?)

**Reply by hermabe** [reply]
I m not familiar with how the angle calculations are done from the IQ samples, but I imagine it is useful to know the channel that was used.

The channel map update in bluetooth is generally used to not interfere with other devices using the same frequency band and not to just send on a single channel.
